docs/faq/lukso/project-timeline.md

```diff
@@ -0,0 +1,34 @@
+---
+sidebar_label: 'Project Timeline'
+sidebar_position: 4
+---
+
+# Project Timeline
+
+### When did LUKSO launch its Testnet?
+
+The LUKSO Testnet was launched on [3 May 2023 at 4:20 PM UTC](https://medium.com/lukso/genesis-validators-deposit-smart-contract-freeze-and-testnet-launch-c5f7b568b1fc). It allowed developers to shift their dApps from L16 and test them before going live on the mainnet. The launch also served as a pre-test for mainnet, which went live only 20 days later. The testnet features the same architecture as the mainnet- however, depositing validators requires being whitelisted.
```

Comment:
do we really need this question?

Reply:
People ask about l16 and when its shut down often. I would suggest keeping it until the end of the year.